### DISCUSSION TITLE
Adding two alert in one

### DISCUSSION THREAD

**Discussion by praveens862** [jertel/elastalert2#859] [Q&A]
Hi team,
I want to make an aler like if from an IP we get accept connection on firewall and within a certain time we get any suspicious activity on endpoint. It is possible in elastalert 2. Please help I have little bit knowledge of elastalert , so plz eloborate the process how I make this rule ,plz suggest.
Thankyou.

**Comment by BuffaloWill** [comment]
The [alert command feature](https://elastalert2.readthedocs.io/en/latest/ruletypes.html?highlight=command#command) can script decisions based on an initial alert.
